src/main/java/net/dv8tion/jda/api/entities/channel/concrete/ForumChannel.java

```diff
@@ -243,6 +251,62 @@ default boolean isTagRequired()
     @CheckReturnValue
     ForumPostAction createForumPost(@Nonnull String name, @Nonnull MessageCreateData message);
 
+    /**
+     * The layout used to sort forum posts.
+     */
+    enum Layout
+    {
+        /**
+         * Displayed as default (not set).
+         */
+        DEFAULT_VIEW(0),
+        /**
+         * Displayed as a chronological list.
+         */
+        LIST_VIEW(1),
+        /**
+         * Displayed as a collection of tiles.
+         */
+        GALLERY_VIEW(2);
+
+        private final int key;
+
+        Layout(int key)
+        {
+            this.key = key;
+        }
+
+        /**
+         * The underlying value as used by Discord.
+         *
+         * @return The raw order key
+         */
+        public int getKey()
+        {
+            return key;
+        }
+
+        /**
+         * The {@link Layout} for the provided key.
+         *
+         * @param  key
+         *         The key to get the {@link Layout} for
+         *
+         * @return The {@link Layout} for the provided key, or {@link #DEFAULT_VIEW} if the key is not known
+         */
+        @Nonnull
+        public static Layout fromKey(int key)
+        {
+            for (Layout layout : values())
+            {
+                if (layout.key == key)
+                    return layout;
+            }
+
+            return DEFAULT_VIEW;
```

Comment:
This should be `UNKNOWN`, which can have key `-1`. Discord might add a new layout, which is not the default view.